packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -204,6 +204,15 @@ export class CdkErrorMapper {
       errorName: 'MultipleSandboxInstancesError',
       classification: 'ERROR',
     },
+    {
+      errorRegex:
+        /InvalidParameterValueException:(.*) (size must be smaller than|exceeds the maximum allowed size of) (?<maxSize>.*) bytes/,
+      humanReadableErrorMessage: 'Maximum Lambda size exceeded',
+      resolutionMessage:
+        'Make sure Lambda code is smaller than {maxSize} bytes unzipped',
```

Comment:
I wonder if we should expose `bundleAwsSdk` and `sourceMap` options for customers to be able to get out of this somehow. With Amplify controlling most of the bundling options, this might not be fixable by customer easily.
```suggestion
        'Make sure you Lambda bundled packages with layers and dependencies is smaller than {maxSize} bytes unzipped. Use `minify` to reduce your lambda package size <Add amplify documentation link>.',
```

Reply:
Yeah if they actually need that much space we may need to expose more bundling options. I'll update the message to what you suggest for now except I see that we don't have docs for `minify`

Reply:
I'll update the message for now excluding `minify` until we have docs for it.